### PR TITLE
Invalid ccg feedback

### PR DESCRIPTION
**Pull request by stephenharland**
Added feedback for GP Practice, CCG and Lead AMHP when no results are returned from typeahead.

### CODE REVIEW

File: Fmas12d.Ui/src/app/components/referral/referral-create/referral-create.component.html [hunk 1]
```diff
@@ -327,6 +327,11 @@ <h6 class="card-subtitle mb-2 text-muted">
                 </div>
               </div>
             </div>
+            <div class="col-md-8 mb-2 mt-less-1" *ngIf="amhpField.errors">
+              <div *ngIf="amhpField.errors.InvalidAmhp" class="invalid-field">
+                <span>* Invalid Lead AMHP </span>
+              </div>
+            </div>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/referral/referral-create/referral-create.component.html]: Having it here makes it misaligned with the other errors.
Try moving the amhpField.errors.InvalidAmhp div under the amhpField.errors.NoMatchingResults div

File: Fmas12d.Ui/src/app/components/referral/referral-edit/referral-edit.component.html [hunk 2]
```diff
@@ -231,6 +231,13 @@ <h6 class="card-subtitle mb-2 text-muted">
                 </div>
               </div>
             </div>
+            <div class="form-row" *ngIf="amhpField.errors">
+              <div class="col-md-12">
+                <div *ngIf="amhpField.errors.InvalidAmhp" class="invalid-field">
+                  <span>* Invalid Lead AMHP </span>
+                </div>
+              </div>
+            </div>
```
Comment (neildavies11) [inline Fmas12d.Ui/src/app/components/referral/referral-edit/referral-edit.component.html]: Same here:
Try moving the amhpField.errors.InvalidAmhp div under the amhpField.errors.NoMatchingResults div

File: Fmas12d.Ui/src/app/components/referral/referral-create/referral-create.component.html (hunk 1 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/referral/referral-create/referral-create.component.html]: moved error message so it is in line with others

File: Fmas12d.Ui/src/app/components/referral/referral-edit/referral-edit.component.html (hunk 2 above)
Comment (stephenharland) [reply Fmas12d.Ui/src/app/components/referral/referral-edit/referral-edit.component.html]: moved error message so it is in line with others

**Pull request merged**